rocAL/source/loaders/image/numpy_loader.cpp

```diff
@@ -154,7 +154,9 @@ NumpyLoader::load_routine() {
     LOG("Started the internal loader thread");
     LoaderModuleStatus last_load_status = LoaderModuleStatus::OK;
     // Initially record number of all the numpy arrays that are going to be loaded, this is used to know how many still there
-    auto max_shape = _output_tensor->info().max_shape();
+    auto max_shape = _output_tensor->info().dims();
+    auto data_layout = _output_tensor->info().layout();
+    max_shape.erase(max_shape.begin());  // Tensor dims contains the batch dimension so removing it to get max_shape
```

Comment:
please use std::copy required elements instead of copying all in line#157. Remove std::erase